frontend/src/routes/_oh.app/hooks/use-ws-status-change.ts

```diff
@@ -51,7 +47,6 @@ export const useWSStatusChange = () => {
     let additionalInfo = "";
 
     if (gitHubToken && selectedRepository) {
-      dispatch(clearSelectedRepository());
```

Comment:
This here was clearing the selected repository. That's why the bottom right it was not showing the repository correctly. Is removing this okay? I tested it but don't know all the impacts.

I also needed to remove this because otherwise "selectedRepository" would be null and so the "Push" buttons would not show up correctly.

Reply:
ugh I was just hitting this too!